src/components/filter-modal/FilterModal.jsx

```diff
@@ -0,0 +1,107 @@
+import { useNotesContext } from "context";
+import "./FilterModal.css";
+
+const FilterModal = ({ setIsFilter }) => {
+  const {
+    notesState: { label, priority, sortByLatest },
+    notesDispatch,
+  } = useNotesContext();
+
+  const listOfLabels = ["office", "food", "home"];
+  const listOfPriorities = ["low", "medium", "high"];
+
+  return (
+    <div className="filter">
+      <div className="flex space-between">
+        <h4>Tags:</h4>
+        <button
+          className="btn btn-clear"
+          onClick={() => {
+            notesDispatch({ type: "RESET_FILTERS" });
+            setIsFilter(false);
+          }}
+        >
+          clear all
+        </button>
+      </div>
+      <div className="flex gap-1rem flex-start">
+        {listOfLabels.map((singleLabel) => (
+          <label
+            key={singleLabel}
```

Comment:
I understand that each of the labels are meant to be unique but just to avoid man made errors, you could try to have ids for each label and using that as key.